README.md

```diff
@@ -43,10 +43,6 @@ On node v0.10 or with file system like HFS or FAT, `fs.stat` time attributes lik
 
 Assuming `lastRun(fn)` returns 1426000001111, `lastRun(fn, 1000)` returns 1426000001000.
```

Comment:
@sttk it seems that the descriptions above can be removed too.

Reply:
Got it. I'll remove the line.